### PR TITLE
Add block prefix to csrf token field

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #...
| License       | MIT
| Doc PR        | https://github.com/symfony/symfony-docs/pull/10867

Currently I use the following code snippet to overwrite the token rendering:

```twig
{%- block hidden_widget -%}
    {%- if form.vars.name == '_token' -%}
        {{ block('app__token_widget') }}
    {%- else -%}
        {{ block('hidden_widget', 'form_div_layout.html.twig') }}
    {%- endif -%}
{%- endblock hidden_widget -%}

{%- block app__token_widget %}
    {{ render_esi(controller('SuluFormBundle:FormWebsite:token', { 'form': form.parent.vars.name })) }}
{%- endblock app__token_widget -%}
```

With the change of https://symfony.com/blog/new-in-symfony-4-3-simpler-form-theming this workaround can now be removed and the following can be used:

```twig
{%- block token_widget %}
    {{ render_esi(controller('SuluFormBundle:FormWebsite:token', { 'form': form.parent.vars.name })) }}
{%- endblock token_widget -%}
```

### CODE REVIEW

File: src/Symfony/Component/Form/Extension/Csrf/Type/FormTypeCsrfExtension.php [hunk 1]
```diff
@@ -90,9 +90,10 @@ public function finishView(FormView $view, FormInterface $form, array $options)
             $tokenId = $options['csrf_token_id'] ?: ($form->getName() ?: \get_class($form->getConfig()->getType()->getInnerType()));
             $data = (string) $options['csrf_token_manager']->getToken($tokenId);
 
-            $csrfForm = $factory->createNamed($options['csrf_field_name'], 'Symfony\Component\Form\Extension\Core\Type\HiddenType', $data, array(
+            $csrfForm = $factory->createNamed($options['csrf_field_name'], 'Symfony\Component\Form\Extension\Core\Type\HiddenType', $data, [
+                'block_prefix' => 'token',
```
Comment (javiereguiluz) [inline src/Symfony/Component/Form/Extension/Csrf/Type/FormTypeCsrfExtension.php]: `token` -> `csrf_token` ?

**Comment by javiereguiluz** [comment]
This looks nice to me ... but I'd like to ask @HeahDude and @vudaltsov if they can imagine any problem for adding this option unconditionally and for "hardcoding" its value. Thanks!

**Review by vudaltsov — APPROVED** [review]

**Review by HeahDude — APPROVED** [review]
This is a really simple and nice addition, thank you very much!

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @alexander-schranz.

**Pull request merged**